overlord/configstate/configcore/ctrlaltdel.go

```diff
@@ -0,0 +1,141 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package configcore
+
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/overlord/configstate/config"
+	"github.com/snapcore/snapd/sysconfig"
+	"github.com/snapcore/snapd/systemd"
+)
+
+// Systemd unit name to control ctrl-alt-del behaviour
+const (
+	ctrlAltDelTarget = "ctrl-alt-del.target"
+)
+
+// Supported actions for core.system.ctrl-alt-del-action
+const (
+	ctrlAltDelReboot = "reboot"
+	ctrlAltDelNone   = "none"
+)
+
+func init() {
+	supportedConfigurations["core.system.ctrl-alt-del-action"] = true
+}
+
+type sysdCtrlAltDelLogger struct{}
+
+func (l *sysdCtrlAltDelLogger) Notify(status string) {
+	fmt.Fprintf(Stderr, "sysd: %s\n", status)
```

Comment:
Unless there's a reason to keep the comment short, I'd rather log this as "CtrlAltDel handler: %s", which might be easier to grep for.

Also, I'm not sure whether you are using `fmt.Fprintf` (instead of `logger.Noticef`) because of a conscious choice (if so, please add a short comment about it), or because you din't know about it.

Reply:
Good catch. I have fixed this here, but I will also push a PR to fix this elsewhere. (Splitting it up to prevent the new PR depend on this patch)

Reply:
Discussed and approved.